x-pack/plugins/uptime/public/lib/alert_types/monitor_status.tsx

```diff
@@ -63,7 +63,9 @@ export const initMonitorStatusAlertType: AlertTypeInitializer = ({
   id: CLIENT_ALERT_TYPES.MONITOR_STATUS,
   name: <MonitorStatusTitle />,
   iconClass: 'uptimeApp',
-  alertParamsExpression: params => <AlertMonitorStatus {...params} autocomplete={autocomplete} />,
+  alertParamsExpression: (params: any) => (
+    <AlertMonitorStatus {...params} autocomplete={autocomplete} />
+  ),
```

Comment:
In this PR we fixed the type we expose here (the type hasn't changed, it's just no longer exposed as an `any` on our end) which revealed that the uptime alert expects `error` to be an `IErrorObject[]` but in our type it's `IErrorObject`.
This isn't a new problem introduced in this PR - this PR has simply revealed an existing problem that was hidden away by the use of `any` on our end.

I'll submit an issue to Uptime to investigate this and migrate to lazy loading if possible.

Reply:
👍